pkg/cmd/server/storage/configmap/storage.go

```diff
@@ -86,3 +86,15 @@ func (c *ConfigMapBridgeStorage) Remove(key string) error {
 	_, err = c.cl.ConfigMaps(c.ns).Update(context.Background(), cm, metav1.UpdateOptions{})
 	return err
 }
+
+func (c *ConfigMapBridgeStorage) List() ([]string, error) {
```

Comment:
With the mocked location/storage servers, are you able to write some unit tests for these new functions?

Reply:
fyi the unit tests I added are up at the pkg/cmd/server/storage layer, where then provision a configmap storage type and then interact with this layer that way